fiftyone/plugins/definitions.py

```diff
@@ -194,6 +202,7 @@ def to_dict(self):
         """
         return {
             "name": self.name,
+            "builtin": self.builtin,
```

Comment:
?

Reply:
I find that moving this arbitrarily adds to the OSS => teams conflicts.

Reply:
I moved it so that `builtin` appears immediately below `name` when printing plugin info:

```
$ fiftyone plugins info @voxel51/operators
key                     value
----------------------  -----------------------------------------------------
name                    @voxel51/operators
builtin                 True
...
```

That's a bit hacky of course as it relies on dicts having a stable iteration order, which isn't necessarily guaranteed in general.

Reply:
+1 for not causing diffs that have no functional purpose though

Reply:
Ah actually this makes sense, just didn't realize.